frontend/packages/data-portal/public/locales/en/translation.json

```diff
@@ -266,6 +275,7 @@
   "runs": "Runs",
   "runsTab": "Runs {{count}}",
   "runsTooltip": "<semibold>Run:</semibold> A tomography run is a collection of all data and annotations related to one physical location in a biological specimen. <url to='/faq?accordion=data-schema'>Learn More</url>",
+  "runsTooltipDepositionSubtext": "Count only shows the number of runs containing deposition data. <url to='/' variant='dashed-underlined' inherit=true>Learn more about depositions in the portal</url>.",
```

Comment:
@kne42 minor nit / not a blocker — please change the gray link text color to match the rest of the gray text instead of blue (helps with hierarchy).

Reply:
weird i thought i fixed this locally but guess not